app/windows/runner/main.cpp

```diff
@@ -8,8 +8,11 @@
 
 int APIENTRY wWinMain(_In_ HINSTANCE instance, _In_opt_ HINSTANCE prev,
                       _In_ wchar_t *command_line, _In_ int show_command) {
-
+#ifdef _DEBUG
+  HWND hwnd = ::FindWindow(L"FLUTTER_RUNNER_WIN32_WINDOW", L"Acter-dev");
```

Comment:
In windows, which stage in compilation will deal with `Acter-dev`?

Reply:
To make sure we use a different window system and don't block the installed version.

Reply:
Then windows and linux will detect release/debug flag using the different logic?

Reply:
this is the windows one, yes.